tests/crypto/src/common_test.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <stdio.h>
+#include <stdarg.h>
+#include <string.h>
+#include <stdint.h>
+#include <stddef.h>
+#include <misc/printk.h>
+
+#include <sys/util.h>
+#include <sys_clock.h>
+
+#include "common_test.h"
+
+volatile uint32_t t_start;
+volatile uint32_t t_stop;
+
+int init_cc310(void)
+{
+    return 0;
+}
```

Comment:
What's the point of this?

Do we intend to support multiple implementations of init_cc310 ?

Reply:
During development, some initialization was necessary. This phased out, and this remained.
Not needed, removed in 7aee7a9